apps/docs/content/guides/database/connection-management.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: 'connection-management'
+title: 'Connection management'
+description: 'Managing connections'
+subtitle: 'Using your connections resourcefully'
+---
+
+## Connections
+
+Every [Compute Add-On](/docs/guides/platform/compute-add-ons) has a pre-configured direct connection count and Supavisor pool size. This guide discusses ways to observe and manage them resourcefully.
+
+## What is a pool
+
+When clients (backend servers) connect to PostgreSQL or Supavisor, they use the PostgreSQL Wire Protocol. Unlike HTTP requests, these connections are stateful and persistent, involving a costly startup but becoming efficient once established.
+
+Clients may create several active connections with Postgres, reducing the overhead of establishing connections between queries.
```

Comment:
This could easily apply to both direct database connections and database connection pool. I would just focus on differentiating how a pool is useful etc.

Reply:
I decided to remove it. This was discussed thoroughly in the connecting-to-postgres docs. Because it was placed away from the primary docs, I thought it would be prudent to add a bit of a refresher.